package.json

```diff
@@ -13,7 +13,7 @@
     "test": "npm run lint && npm run test:karma",
     "test:karma": "karma start --single-run --browsers ChromeHeadless karma.conf.js",
     "lint": "eslint src test",
-    "build": "rollup -c",
+    "build": "./node_modules/.bin/rollup -c",
```

Comment:
This isn't required per https://docs.npmjs.com/cli/run-script:

![image](https://user-images.githubusercontent.com/45576380/64703052-aec83b00-d479-11e9-9a24-ca4b743debd4.png)

None of the utilities referenced in `scripts` is installed globally (`rollup`, `karma`, `eslint`)

I like the enhanced readability of not including the `./node_modules/.bin/` path.

Would it be a good idea to leave it out?

Reply:
Oh, I wasn't aware of that! I tried running it and it failed, though that was likely user error. 😄 

> I like the enhanced readability of not including the ./node_modules/.bin/ path.

Totally agree - it's cleaner to not have the path.